oneview/resource_hypervisor_cluster_profile.go

```diff
@@ -442,6 +487,7 @@ func resourceHypervisorClusterProfile() *schema.Resource {
 			"state": {
 				Type:     schema.TypeString,
 				Optional: true,
```

Comment:
i think it should be set as computed: true no optional

Reply:
its not a read only field for HCP. I checked in Rest guide.